### PR TITLE
Updating the Readme with the new release

### PR DESCRIPTION
**Pull request by vincentpierre**
### Proposed change(s)

Updating the readme on master

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

**Comment by vincentpierre** [comment]
It breaks as expected since the release tag for 0.15.0 is not out yet.

File: README.md [hunk 1]
```diff
@@ -62,7 +62,8 @@ details of the changes between versions.
 | **Version** | **Release Date** | **Source** | **Documentation** | **Download** |
 |:-------:|:------:|:-------------:|:-------:|:------------:|
 | **master** (unstable) | -- | [source](https://github.com/Unity-Technologies/ml-agents/tree/master) |  [docs](https://github.com/Unity-Technologies/ml-agents/tree/master/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/master.zip) |
-| **0.14.1** (latest stable release) | February 26, 2020 | **[source](https://github.com/Unity-Technologies/ml-agents/tree/latest_release)** |  **[docs](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Readme.md)** | **[download](https://github.com/Unity-Technologies/ml-agents/archive/latest_release.zip)** |
+| **0.15.0** | March 18, 2020 | **[source](https://github.com/Unity-Technologies/ml-agents/tree/latest_release)** |  **[docs](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Readme.md)** | **[download](https://github.com/Unity-Technologies/ml-agents/archive/latest_release.zip)** |
```
Comment (surfnerd) [inline README.md]: I thought the plan was to make this a direct link to the release tag commit from now on.  Is that not the case anymore?

File: README.md [hunk 2]
```diff
@@ -62,7 +62,8 @@ details of the changes between versions.
...
+| **0.14.1**  | February 26, 2020 | [source](https://github.com/Unity-Technologies/ml-agents/tree/0.14.1) |  [docs](https://github.com/Unity-Technologies/ml-agents/tree/0.14.1/docs/Readme.md) | [download](https://github.com/Unity-Technologies/ml-agents/archive/0.14.1) |
```
Comment (mmattar) [inline README.md]: missing .zip in the download link

Comment (vincentpierre) [reply README.md]: I feel like CI should have caught this

Comment (chriselion) [reply README.md]: CI only checks remote links daily, not on PRs. This was due to some links being flaky (often Unity blog posts).

If you want to try messing with regexes to scan `https://github.com` on PRs, they're here:
https://github.com/Unity-Technologies/ml-agents/blob/master/markdown-link-check.fast.json (runs on PRs)
https://github.com/Unity-Technologies/ml-agents/blob/master/markdown-link-check.full.json (runs nightly)

There's also a script to generate the table: https://github.com/Unity-Technologies/ml-agents/blob/master/utils/make_readme_table.py

**Pull request merged**